### DISCUSSION TITLE
Podman 4.2.1 to Podman 5.1.1 - Kernel/yocto/library changes

### DISCUSSION THREAD

**Discussion by msom-apex** [ajayd-san/gomanagedocker#35] [Q&A]
Hi Podman team,
It was overwhelming to see such a huge information about developments and bug fixes since 4.1.2
I would like to explore the latest stable 5.1.1 but am concerned about any changes that needs to be made wrt time and dependencies.

On a high level, here are the concernes:
1. Linux version : Do any component in 5.1.1 need a specific or higher linux version than arm 4.14?
2. Yocto version : Can we still work with thud ranch of yocto? do we need a different one? details on the dependency if yes
3. Libraries: What are the libraries required for the latest podman? 
4. Database : Is a database compulsory to get on with 5.1.1? is yes, what is the minimum that can be adopted? space and other dependencies of the database
5. Communication modification: OCI instead of HTTP: can we still use older method as in 4.2.1 ? if no, what are the dependencies for this?

**Comment by ajayd-san** [comment]
I'm very flattered that you think I wrote podman but...unfortunately not 🤧

**Reply by msom-apex** [reply]
Could you add relevent references that clarify the same please?

**Reply by ajayd-san** [reply]
Dude this the wrong repo...
